models/articleModel.js

```diff
@@ -0,0 +1,45 @@
+const db = require('../db/connection');
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+exports.selectArticleById = (id) => {
+
+  return db.query(`SELECT author, title, article_id, body, topic, created_at, votes, article_img_url
+                  FROM articles
+                  WHERE article_id = $1`, [id])
+    .then((result) => result.rows);
```

Comment:
this result.rows is an array. we want to send back an object

Reply:
And I am sending back an object in the controller lines 28+: 
.then((result) => {
      response.status(200).send({ article: result });
    });
    
on a specified key `article` ?